lib/modules/manager/regex/readme.md

```diff
@@ -1,24 +1,26 @@
-The `regex` manager is designed to allow users to manually configure Renovate for how to find dependencies that aren't detected by the built-in package managers.
+With the `regex` manager you can configure Renovate so it finds dependencies that are not detected by its built-in package managers.
```

Comment:
Admittedly, in nitpicking mode: its **other** built-in package managers.

Reply:
```suggestion
With the `regex` manager you can configure Renovate so it finds dependencies that are not detected by its other built-in package managers.
```